src/query/storages/parquet/src/parquet2/parquet_table/partition.rs

```diff
@@ -159,7 +160,7 @@ impl Parquet2Table {
                 &file_locations,
                 ctx.get_settings().get_max_threads()? as usize,
                 &ctx.get_copy_status(),
-                ctx.get_query_kind().eq_ignore_ascii_case("copy"),
+                ctx.get_query_kind() == QueryKind::Copy,
```

Comment:
The convention for rust is to compare enum with `matches!`

Reply:
done